packages/modules/itwin-viewer-react/src/config/rpc.ts

```diff
@@ -20,6 +23,19 @@ const getSupportedRpcs = () => {
   ];
 };
 
-export const initRpc = (rpcParams: BentleyCloudRpcParams) => {
-  BentleyCloudRpcManager.initializeClient(rpcParams, getSupportedRpcs());
+export const initRpc = (
+  rpcParams: BentleyCloudRpcParams | ElectronRpcParams,
+  isDesktop?: boolean
+) => {
+  if (isDesktop) {
+    ElectronRpcManager.initializeClient(
+      rpcParams as ElectronRpcParams,
+      getSupportedRpcs()
+    );
+  } else {
+    BentleyCloudRpcManager.initializeClient(
+      rpcParams as BentleyCloudRpcParams,
+      getSupportedRpcs()
```

Comment:
should the `SnapshotIModelRpcInterface` be a supported rpc for the web?

Reply:
These are the typical defaults that are used (we support it in IT1), so it shouldn't be an issue. Longer term these will be configurable

Reply:
We do probably want to avoid hosted backends that support Snapshots, at least in the way this method currently handles them.  The ability to pass a file path and open whatever file is there is a security issue.  However, registering it on the frontend and then not using it is fairly harmless.